### DISCUSSION TITLE
Feature Request: time series gap-fill operations

### DISCUSSION THREAD

**Discussion by dshemetov** [duckdb/duckdb#6094] [Ideas]
Any plans to implement a gap-filling function for time series? My current use case is making contiguous time series index out of an irregularly sampled time index. Bucketing isn't exactly what I need, but this seems related [TimescaleDB's `time_bucket_gapfill`](https://github.com/timescale/timescaledb/blob/main/tsl/src/nodes/gapfill/README.md), (maybe natural now that #5665 has been implemented).

**Comment by ayuhito** [comment]
Would love a feature like this as it's very useful with irregular datasets.
